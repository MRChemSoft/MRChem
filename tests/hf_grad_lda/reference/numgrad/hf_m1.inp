world_prec = 1.0e-6                 # Overall relative precision
world_size = 6                      # Size of simulation box 2^n
world_unit = angstrom

# use Psi4 naming of displacements
# displacement 3 of 5 (i.e. point -1 in the stencil) 
Molecule {
$coords
F           -1.194939558201    -0.244647811546     0.341050691237
H           -1.671165086722    -0.394660569802     1.105156770557
$end
translate = false
}

WaveFunction {
  method = LDA                      # Wave function method (HF or DFT)
}

SCF {
  guess_type = CORE_DZ             # Type of initial guess: none, mw, gto
}
