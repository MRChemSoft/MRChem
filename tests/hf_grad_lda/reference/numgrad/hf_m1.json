{
  "input": {
    "molecule": {
      "charge": 0,
      "coords": [
        {
          "atom": "f",
          "xyz": [
            -2.258108500481206,
            -0.4623173608109983,
            0.6444924010522362
          ]
        },
        {
          "atom": "h",
          "xyz": [
            -3.158044322941054,
            -0.7458003891145317,
            2.088443621128611
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -6
    },
    "printer": {
      "file_name": "hf_m1.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": false,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "svwn5"
              }
            ],
            "spin": false
          }
        }
      },
      "initial_guess": {
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT (SVWN5)",
        "prec": 0.001,
        "restricted": true,
        "type": "core",
        "zeta": 2
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "energy_thrs": -1.0,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 5,
        "localize": false,
        "max_iter": 100,
        "method": "DFT (SVWN5)",
        "orbital_thrs": 9.999999999999999e-06,
        "rotation": 0,
        "start_prec": 1e-06
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        -2.303448186190995,
        -0.47659952525713567,
        0.7172401380181691
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.7036316315162943,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -0.3671092048873703,
            -0.11564090271401817,
            0.5890292744649726
          ],
          "vector_el": [
            23.113911622335273,
            4.79101573375509,
            -7.299845956159415
          ],
          "vector_nuc": [
            -23.481020827222643,
            -4.906656636469108,
            7.888875230624388
          ]
        }
      },
      "geometric_derivative": {
        "electronic": [
          0.0,
          0.0,
          0.0
        ],
        "electronic_norm": 0.0,
        "nuclear": [
          0.0,
          0.0,
          0.0
        ],
        "nuclear_norm": 0.0,
        "total": [
          0.0,
          0.0,
          0.0
        ],
        "total_norm": 0.0
      },
      "geometry": [
        {
          "symbol": "F",
          "xyz": [
            -2.258108500481206,
            -0.4623173608109983,
            0.6444924010522362
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            -3.158044322941054,
            -0.7458003891145317,
            2.088443621128611
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -24.087561049500227,
          -1.0984436173286518,
          -0.49981495621782684,
          -0.3612930712519925,
          -0.3612930436002308
        ],
        "occupation": [
          2.0,
          2.0,
          2.0,
          2.0,
          2.0
        ],
        "spin": [
          "p",
          "p",
          "p",
          "p",
          "p"
        ],
        "sum_occupied": -52.816811475797856
      },
      "scf_energy": {
        "E_ee": 55.480310385723044,
        "E_eext": 0.0,
        "E_el": -105.06952138938992,
        "E_en": -249.83936661777568,
        "E_kin": 99.43100557947638,
        "E_next": 0.0,
        "E_nn": 5.217726899833345,
        "E_nuc": 5.217726899833345,
        "E_tot": -99.85179448955657,
        "E_x": 0.0,
        "E_xc": -10.141470736813677
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 20,
      "routine": "mrchem.x",
      "total_cores": 20,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 94.59047423207456,
        "E_eext": 0.0,
        "E_el": -83.26342955730186,
        "E_en": -329.290075173849,
        "E_kin": 166.74359336616132,
        "E_next": 0.0,
        "E_nn": 5.217726899833345,
        "E_nuc": 5.217726899833345,
        "E_tot": -78.04570265746852,
        "E_x": 0.0,
        "E_xc": -15.307421981688737
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 58.07926534367104,
              "E_eext": 0.0,
              "E_el": -104.44876163312424,
              "E_en": -247.25891343409884,
              "E_kin": 95.0293618497547,
              "E_next": 0.0,
              "E_nn": 5.217726899833345,
              "E_nuc": 5.217726899833345,
              "E_tot": -99.2310347332909,
              "E_x": 0.0,
              "E_xc": -10.298475392451104
            },
            "energy_total": -99.2310347332909,
            "energy_update": 21.185332075822373,
            "mo_residual": 1.1964034979067808,
            "wall_time": 24.478675806
          },
          {
            "energy_terms": {
              "E_ee": 50.87087064932163,
              "E_eext": 0.0,
              "E_el": -104.7647795863593,
              "E_en": -239.54595247322493,
              "E_kin": 93.42540495852518,
              "E_next": 0.0,
              "E_nn": 5.217726899833345,
              "E_nuc": 5.217726899833345,
              "E_tot": -99.54705268652594,
              "E_x": 0.0,
              "E_xc": -9.515102720981181
            },
            "energy_total": -99.54705268652594,
            "energy_update": 0.3160179532350469,
            "mo_residual": 0.49753314220075234,
            "wall_time": 22.434449359
          },
          {
            "energy_terms": {
              "E_ee": 57.26652953048719,
              "E_eext": 0.0,
              "E_el": -105.01411682226912,
              "E_en": -252.01076482512985,
              "E_kin": 100.06726028440947,
              "E_next": 0.0,
              "E_nn": 5.217726899833345,
              "E_nuc": 5.217726899833345,
              "E_tot": -99.79638992243576,
              "E_x": 0.0,
              "E_xc": -10.337141812035936
            },
            "energy_total": -99.79638992243576,
            "energy_update": 0.24933723590982027,
            "mo_residual": 0.36051296680142386,
            "wall_time": 22.270986245
          },
          {
            "energy_terms": {
              "E_ee": 54.96365844978841,
              "E_eext": 0.0,
              "E_el": -105.05444459558103,
              "E_en": -248.4214105916094,
              "E_kin": 98.46347143212033,
              "E_next": 0.0,
              "E_nn": 5.217726899833345,
              "E_nuc": 5.217726899833345,
              "E_tot": -99.8367176957477,
              "E_x": 0.0,
              "E_xc": -10.060163885880385
            },
            "energy_total": -99.8367176957477,
            "energy_update": 0.040327773311929604,
            "mo_residual": 0.12658141605357273,
            "wall_time": 22.609853064
          },
          {
            "energy_terms": {
              "E_ee": 55.78111300546398,
              "E_eext": 0.0,
              "E_el": -105.06736407818342,
              "E_en": -250.42698182180231,
              "E_kin": 99.75635186041494,
              "E_next": 0.0,
              "E_nn": 5.217726899833345,
              "E_nuc": 5.217726899833345,
              "E_tot": -99.84963717835006,
              "E_x": 0.0,
              "E_xc": -10.177847122260028
            },
            "energy_total": -99.84963717835006,
            "energy_update": 0.012919482602370636,
            "mo_residual": 0.0778404656307283,
            "wall_time": 23.886737531
          },
          {
            "energy_terms": {
              "E_ee": 55.452114616992546,
              "E_eext": 0.0,
              "E_el": -105.06923309354882,
              "E_en": -249.80802233841322,
              "E_kin": 99.42398725022532,
              "E_next": 0.0,
              "E_nn": 5.217726899833345,
              "E_nuc": 5.217726899833345,
              "E_tot": -99.85150619371547,
              "E_x": 0.0,
              "E_xc": -10.137312622353473
            },
            "energy_total": -99.85150619371547,
            "energy_update": 0.001869015365400628,
            "mo_residual": 0.02188275308630292,
            "wall_time": 23.990137909
          },
          {
            "energy_terms": {
              "E_ee": 55.478212833158864,
              "E_eext": 0.0,
              "E_el": -105.06950439421253,
              "E_en": -249.8486804778999,
              "E_kin": 99.44241111479695,
              "E_next": 0.0,
              "E_nn": 5.217726899833345,
              "E_nuc": 5.217726899833345,
              "E_tot": -99.85177749437918,
              "E_x": 0.0,
              "E_xc": -10.141447864268446
            },
            "energy_total": -99.85177749437918,
            "energy_update": 0.0002713006637122817,
            "mo_residual": 0.009582412228320515,
            "wall_time": 24.472407305
          },
          {
            "energy_terms": {
              "E_ee": 55.48202548963404,
              "E_eext": 0.0,
              "E_el": -105.06951854055413,
              "E_en": -249.84482386981355,
              "E_kin": 99.43500531511522,
              "E_next": 0.0,
              "E_nn": 5.217726899833345,
              "E_nuc": 5.217726899833345,
              "E_tot": -99.85179164072079,
              "E_x": 0.0,
              "E_xc": -10.141725475489817
            },
            "energy_total": -99.85179164072079,
            "energy_update": 1.4146341612786273e-05,
            "mo_residual": 0.0011656556583999062,
            "wall_time": 24.390821599
          },
          {
            "energy_terms": {
              "E_ee": 55.48044324708473,
              "E_eext": 0.0,
              "E_el": -105.06952114390972,
              "E_en": -249.83725419498597,
              "E_kin": 99.42874152720476,
              "E_next": 0.0,
              "E_nn": 5.217726899833345,
              "E_nuc": 5.217726899833345,
              "E_tot": -99.85179424407639,
              "E_x": 0.0,
              "E_xc": -10.141451723213237
            },
            "energy_total": -99.85179424407639,
            "energy_update": 2.60335559687519e-06,
            "mo_residual": 0.0006931385911610831,
            "wall_time": 23.618759967
          },
          {
            "energy_terms": {
              "E_ee": 55.48009593311731,
              "E_eext": 0.0,
              "E_el": -105.06952132658114,
              "E_en": -249.8376707889974,
              "E_kin": 99.42947714684577,
              "E_next": 0.0,
              "E_nn": 5.217726899833345,
              "E_nuc": 5.217726899833345,
              "E_tot": -99.8517944267478,
              "E_x": 0.0,
              "E_xc": -10.14142361754683
            },
            "energy_total": -99.8517944267478,
            "energy_update": 1.826714139951946e-07,
            "mo_residual": 0.0001781120229753016,
            "wall_time": 25.325824879
          },
          {
            "energy_terms": {
              "E_ee": 55.480368240432455,
              "E_eext": 0.0,
              "E_el": -105.06952138470494,
              "E_en": -249.83968769545208,
              "E_kin": 99.43128266515573,
              "E_next": 0.0,
              "E_nn": 5.217726899833345,
              "E_nuc": 5.217726899833345,
              "E_tot": -99.85179448487159,
              "E_x": 0.0,
              "E_xc": -10.14148459484103
            },
            "energy_total": -99.85179448487159,
            "energy_update": 5.8123788448938285e-08,
            "mo_residual": 9.354878903239376e-05,
            "wall_time": 24.7883954
          },
          {
            "energy_terms": {
              "E_ee": 55.48028677209104,
              "E_eext": 0.0,
              "E_el": -105.06952138817296,
              "E_en": -249.83955871598366,
              "E_kin": 99.43122410981526,
              "E_next": 0.0,
              "E_nn": 5.217726899833345,
              "E_nuc": 5.217726899833345,
              "E_tot": -99.8517944883396,
              "E_x": 0.0,
              "E_xc": -10.141473554095588
            },
            "energy_total": -99.8517944883396,
            "energy_update": 3.468016984697897e-09,
            "mo_residual": 2.016720112578475e-05,
            "wall_time": 25.132541701
          },
          {
            "energy_terms": {
              "E_ee": 55.48035190223536,
              "E_eext": 0.0,
              "E_el": -105.06952138912536,
              "E_en": -249.83954181313115,
              "E_kin": 99.431146816346,
              "E_next": 0.0,
              "E_nn": 5.217726899833345,
              "E_nuc": 5.217726899833345,
              "E_tot": -99.85179448929202,
              "E_x": 0.0,
              "E_xc": -10.141478294575563
            },
            "energy_total": -99.85179448929202,
            "energy_update": 9.524114830128383e-10,
            "mo_residual": 1.7777040086636097e-05,
            "wall_time": 24.707380233
          },
          {
            "energy_terms": {
              "E_ee": 55.480310385723044,
              "E_eext": 0.0,
              "E_el": -105.06952138938992,
              "E_en": -249.83936661777568,
              "E_kin": 99.43100557947638,
              "E_next": 0.0,
              "E_nn": 5.217726899833345,
              "E_nuc": 5.217726899833345,
              "E_tot": -99.85179448955657,
              "E_x": 0.0,
              "E_xc": -10.141470736813677
            },
            "energy_total": -99.85179448955657,
            "energy_update": 2.645492713782005e-10,
            "mo_residual": 7.942816204825182e-06,
            "wall_time": 24.703511272
          }
        ],
        "wall_time": 336.812348379
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
