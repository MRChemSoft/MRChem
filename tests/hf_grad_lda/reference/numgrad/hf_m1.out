                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         mrchem-gradient                          ***
***         Git commit hash    c6b193dd4223e5895077                     ***
***         Git commit author  Roberto Di Remigio                       ***
***         Git commit date    Fri Sep 25 22:32:15 2020 +0200           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                              20
 Total cores             :                                              20
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.3.1
 Git branch            : mrchem-gradient
 Git commit hash       : b79bdf06c3c82a53edb6
 Git commit author     : Roberto Di Remigio
 Git commit date       : Tue Sep 15 18:31:50 2020 +0200

 Linear algebra        : EIGEN v3.3.7
 Parallelization       : MPI/OpenMP

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   64.00000    64.00000    64.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -64.00000   -64.00000   -64.00000 ]
 upper bounds          : [   64.00000    64.00000    64.00000 ]
 total length          : [  128.00000   128.00000   128.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.258109       -0.462317        0.644492
    1       H            :       -3.158044       -0.745800        2.088444
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize Core Hamiltonian matrix
 Precision               : 1.00000e-03
 Restricted              : True
 AO basis                : Hydrogenic orbitals
 Zeta quality            : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               5
 Beta electrons          :                                               5
 Total electrons         :                                              10
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.999932195761e-01
    1    2    p          :                              9.999513046978e-01
    2    2    p          :                              9.999999998818e-01
    3    2    p          :                              9.999999964368e-01
    4    2    p          :                              9.999999870029e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (SVWN5)
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                166.743593366161
 E-N energy              :            (au)               -329.290075173849
 Coulomb energy          :            (au)                 94.590474232075
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -15.307421981689
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  5.217726899833
 Ext. field (nuc)        :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -83.263429557302
 Nuclear energy          :            (au)                  5.217726899833
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.804570265747e+01
                         :      (kcal/mol)             -4.897441743232e+04
                         :        (kJ/mol)             -2.049089626698e+05
                         :            (eV)             -2.123731666510e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -14.609867175214
    1    2    p          :            (au)                  3.223797885188
    2    2    p          :            (au)                  4.746104044964
    3    2    p          :            (au)                  4.772228944227
    4    2    p          :            (au)                  5.076787341479
---------------------------------------------------------------------------
 Sum occupied            :            (au)                  6.418102081288
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (SVWN5)
 Checkpointing           : Off
 Max iterations          : 100
 KAIN solver             : 5
 Localization            : Off
 Diagonalization         : First two iterations
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : Off
 Orbital threshold       : 1.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          2.236068e+00         -78.045702657469         -7.804570e+01
    1          1.196403e+00         -99.231034733291         -2.118533e+01
    2          4.975331e-01         -99.547052686526         -3.160180e-01
    3          3.605130e-01         -99.796389922436         -2.493372e-01
    4          1.265814e-01         -99.836717695748         -4.032777e-02
    5          7.784047e-02         -99.849637178350         -1.291948e-02
    6          2.188275e-02         -99.851506193715         -1.869015e-03
    7          9.582412e-03         -99.851777494379         -2.713007e-04
    8          1.165656e-03         -99.851791640721         -1.414634e-05
    9          6.931386e-04         -99.851794244076         -2.603356e-06
   10          1.781120e-04         -99.851794426748         -1.826714e-07
   11          9.354879e-05         -99.851794484872         -5.812379e-08
   12          2.016720e-05         -99.851794488340         -3.468017e-09
   13          1.777704e-05         -99.851794489292         -9.524115e-10
   14          7.942816e-06         -99.851794489557         -2.645493e-10
---------------------------------------------------------------------------
                      SCF converged in 14 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.258109       -0.462317        0.644492
    1       H            :       -3.158044       -0.745800        2.088444
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                 99.431005579476
 E-N energy              :            (au)               -249.839366617776
 Coulomb energy          :            (au)                 55.480310385723
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -10.141470736814
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  5.217726899833
 Ext. field (nuc)        :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -105.069521389390
 Nuclear energy          :            (au)                  5.217726899833
---------------------------------------------------------------------------
 Total energy            :            (au)             -9.985179448956e+01
                         :      (kcal/mol)             -6.265794653884e+04
                         :        (kJ/mol)             -2.621608484886e+05
                         :            (eV)             -2.717105627789e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -24.087561049500
    1    2    p          :            (au)                 -1.098443617329
    2    2    p          :            (au)                 -0.499814956218
    3    2    p          :            (au)                 -0.361293071252
    4    2    p          :            (au)                 -0.361293043600
---------------------------------------------------------------------------
 Sum occupied            :            (au)                -52.816811475798
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       23.113912        4.791016       -7.299846
 Magnitude               :            (au)                       24.708187
                         :         (Debye)                       62.801941
---------------------------------------------------------------------------
 Nuclear vector          :      -23.481021       -4.906657        7.888875
 Magnitude               :            (au)                       25.252088
                         :         (Debye)                       64.184401
---------------------------------------------------------------------------
 Total vector            :       -0.367109       -0.115641        0.589029
 Magnitude               :            (au)                        0.703632
                         :         (Debye)                        1.788453
===========================================================================


===========================================================================
                           Geometric Derivative
---------------------------------------------------------------------------
 Total                   :        0.000000        0.000000        0.000000
 Norm                    :            (au)                        0.000000
---------------------------------------------------------------------------
 Nuclear                 :        0.000000        0.000000        0.000000
 Norm                    :            (au)                        0.000000
---------------------------------------------------------------------------
 Electronic              :        0.000000        0.000000        0.000000
 Norm                    :            (au)                        0.000000
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  5m 43s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
